market/connector/adobe-esign-connector/meta.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://json-schema.axonivy.com/market/10.0.0/meta.json",
+    "id": "adobe-esign-connector",
+    "name": "Adobe Esign Connector",
+    "description": "Use this connector to create, sign document with Adobe Sign.",
```

Comment:
Please reword to:
Use this connector to send, sign, track, and manage signature processes in Axon Ivy